opennms-services/src/main/java/org/opennms/netmgt/vacuumd/Vacuumd.java

```diff
@@ -316,7 +316,11 @@ public Scheduler getScheduler() {
 
     private void scheduleAutomations() {
         for (Automation auto : getVacuumdConfig().getAutomations()) {
-            scheduleAutomation(auto);
+            try {
+                scheduleAutomation(auto);
+            } catch (Exception e) {
+                LOG.warn("Could not schedule automation {}", auto, e);
```

Comment:
We might want to just log the name rather than the whole `Automation` object. What do you think. If the `Automation` `toString()` is going to be readable we can keep it as is, otherwise switch it up here.

Reply:
I agree. Changed.

Reply:
We could possibly NPE dereferencing `auto` now. I think it is ok, we should be able to assume we don't get given any null automations and if we do get given a null automation that is likely because something is really wrong and preventing startup is likely OK in that case.